lib/abci/handlers/endBlockHandlerFactory.js

```diff
@@ -0,0 +1,37 @@
+const {
+  abci: {
+    ResponseEndBlock,
+  },
+} = require('abci/types');
+
+/**
+ * End block ABCI handler
+ *
+ * @param {BlockchainState} blockchainState
+ * @param {BaseLogger} logger
+ *
+ * @return {endBlockHandler}
+ */
+function endBlockHandlerFactory(
```

Comment:
This is out of scope of this task. Here we just need to obtain ChainLock and store it in LatestCoreChainLock class

Reply:
Another task required endBlock and this PR is built on top of that.

Reply:
Yeah. Probably we could just remove handlers from this branch? :))

Reply:
Removed !